### DISCUSSION TITLE
Multiple Phishing Sites Takedown Request Follow-up

### DISCUSSION THREAD

**Discussion by e-tiny-project** [vercel/community#6908] [Help] [answered]
### Summary

Hi, 

Requesting for a follow-up on multiple phishing pages that are running through your services. 
Please help shut down these pages as they continue to lure unsuspecting users to connect their crypto wallets and drain their funds. 

Already submitted reports using abuse@vercel.com last week. Here are the case numbers:

00230976
00230975
00230973
00230970
00230968
00230964
00230961
00230960
00230955
00230957
00200587

Thanks in advance. 

### Steps to Reproduce

Go to the case number report and open the URL reported.

**Comment by milannshrestha** [comment]
cc @amyegan

**Comment by amyegan** [comment, marked as answer]
Thanks for letting us know about pages you believe to be problematic. You've already done the right thing by submitting your reports to [abuse@vercel.com](mailto:abuse@vercel.com), and I see that you already received a response on some of them. Please follow up with your open cases if you have any questions or additional information to share.

**Reply by milannshrestha** [reply]
Hi @amyegan , We appreciate your response, but the cases above are **ACTIVE** and has been causing business impact. We seek your help and resource to take down abusive domains under Vercel.

**Reply by e-tiny-project** [reply]
Hello @amyegan , I've already sent a follow-up with each open case below; only 2 of them have been closed. Please assist us in getting these pages disabled as soon as possible. 

00230976
00230975
00230973 -  Closed
00230970 
00230968 -  Closed
00230964
00230961
00230960
00230955
00230957
00200587

Thank you
